src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.ts

```diff
@@ -145,7 +145,7 @@ export class RgwMultisiteDetailsComponent implements OnDestroy, OnInit {
 
   openModal(entity: any | string, edit = false) {
     const entityName = edit ? entity?.data?.type : entity;
-    const action = edit ? 'edit' : 'create';
+    const action = edit ? 'Edit' : 'Create';
```

Comment:
you can use the `actionLabels` from here itself right and then avoid the extra if condition in the form component.